react-native/update-font-icon.js

```diff
@@ -0,0 +1,44 @@
+// Generate our react-native icons from the fonticon stylesheet and images
+import fs from 'fs'
+import path from 'path'
+
+const css = fs.readFileSync('../desktop/renderer/fonticon.css', {encoding: 'utf8'})
+const stream = fs.createWriteStream('../shared/common-adapters/icon.paths.native.js')
+
+stream.once('open', () => {
+  stream.write('// This file is generated by update-font-icon.js, DO NOT HAND EDIT!\n')
+  stream.write('// @flow\n')
+  stream.write('\n')
+
+  stream.write('export const fontIcons = {\n')
+
+  const glyphs = []
+  let glyph = null
+  css.split('\n').forEach(line => {
+    if (glyph) {
+      const val = line.match(/content:\ \"\\(.*)\";/)[1]
```

Comment:
i think it'll be hard to detect this accurately. the css file itself is generated so its structure is well known (now). if that changes in any way we'll have to rewrite this thing anyways. i don't think we can expand this thing in a useful way now that'll make it understand when there are , like more matches or something. this will crash (on purpose) if there isn't a match value if you've captured a glyph